django/chat/models.py

```diff
@@ -289,7 +290,7 @@ def sorted_files(self):
 
     @property
     def sources(self):
-        return self.answersource_set.all()
+        return self.answersource_set.all().order_by("id")
```

Comment:
On a seemingly random basis, the bulk creation of AnswerSource objects would reorder the sources. I assume this has something to do with the async'ing, because the Django docs say that bulk_create should retain the same order as the input list. This fix ensures that the order established in qa_response is same as their display order in the modal.

Reply:
Well, it does retain the same order (of IDs) :)

Reply:
That still appears to be inconsistent with the stated behaviour of bulk_create. Django docs suggest that the resulting objects will retain the order of the input list, but after running this line in utils.save_sources_and_update_security_label:

`AnswerSource.objects.bulk_create(sources)`

I'd randomly see sources (the input list) ordered differently from the resulting message.sources (the created model instances). This stopped happening when I introduced the explicit ordering above. Aside from a possible error in the Django source code, async'ing is the only thing I can think of that'd cause this - and it does appear to be "genuinely" random.